commit f43731fd344cd3747c0fd7b10b1a694cacf530c4
Author: Sage Weil <sweil@redhat.com>
Date:   Tue Jun 12 11:44:16 2018 -0500

    msg: make set_addr_unknowns take an addrvec
    
    Nicely collapse the wonky logic in OSD into the helper!
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/DaemonServer.cc b/src/mgr/DaemonServer.cc
index f7feae2ea20..257177ea156 100644
--- a/src/mgr/DaemonServer.cc
+++ b/src/mgr/DaemonServer.cc
@@ -134,7 +134,7 @@ int DaemonServer::init(uint64_t gid, entity_addr_t client_addr)
   }
 
   msgr->set_myname(entity_name_t::MGR(gid));
-  msgr->set_addr_unknowns(client_addr);
+  msgr->set_addr_unknowns(entity_addrvec_t(client_addr));
 
   msgr->start();
   msgr->add_dispatcher_tail(this);
